pkgs/applications/misc/waybar/default.nix

```
@@ -71,19 +71,19 @@ let
   libcava.src = fetchFromGitHub {
     owner = "LukashonakV";
     repo = "cava";
-    rev = "0.10.1";
+    rev = "0.10.2";
```

```suggestion
    rev = "0.10.1";
```

I don't think changing this is necessary/relevant

---

Upstream is still using 0.10.1 https://github.com/Alexays/Waybar/blob/master/subprojects/cava.wrap